src/Azs.ContainerRegistry.Admin/docs/Set-AzsContainerRegistryConfiguration.md

```diff
@@ -33,7 +32,7 @@ Configure container registry overall configuration properties.
 
 ### Example 1: Set Azs ContainerRegistry Configuration
 ```powershell
-PS C:\> Set-AzsContainerRegistryConfiguration -MaximumCapacityInGib 30 -ConfigurationName default | ConvertTo-Json
+PS C:\> Set-AzsContainerRegistryConfiguration -MaximumCapacityInGib 30 | ConvertTo-Json
```

Comment:
how are you setting the default value for ConfigurationName

Reply:
The parameter is optional, so it also works if it is presented. Default value is added in readme.md:
    # Rename cmdlet parameter name in ContainerRegistryConfiguration
  - where:
      subject: ContainerRegistryConfiguration
      parameter-name: ConfigurationName
    set:
      parameter-name: Name
  - where:
      verb: Remove
      subject: ContainerRegistryConfiguration
      parameter-name: Name
    set:
      default:
        script: Write-Output "default"
  - where:
      verb: Set
      subject: ContainerRegistryConfiguration
      parameter-name: Name
    set:
      default:
        script: Write-Output "default"